### ISSUE TITLE
Refine the design for the state dashboard

### ISSUE THREAD

**Issue by lauraponce** [18F/cms-hitech-apd#676]
Refer back to #383 to continue refining the design of the state dashboard 
### This task is done when...
 - [ ] v3 has been wireframed
 - [ ] design has been shared with the team for feedback

**Comment by lauraponce** [comment]
![state dashboard v3](https://user-images.githubusercontent.com/21222704/41759594-17916a02-75a4-11e8-909e-77b9b83bee81.png)

**Comment by lauraponce** [comment]
Updates in this version: 

- Added tasks section to draw attention to action items 
- Cleaned up statuses section (added link to approval letter and made call to action more prominent)
- Refined the style for the recent activity section
- Removed the calendar and changed the events section to a list format. This is where users would see upcoming important dates, deadlines, events, etc. (also added option for adding their own event here)
- Matched the overall design to the new styles

**Comment by lauraponce** [comment]
@NAretakis @jeromeleecms any feedback on this? @quarterback can you review this content pretty please? cc @nkkl  

I ran this through my design crit group, so some of these changes reflect their suggestions.

**Comment by nkkl** [comment]
This is sick, Laura.

**Comment by jeromeleecms** [comment]
@lauraponce -- I like the look and feel overall. But thinking through the views, there may be pieces to noodle over:

- I'm feeling a bit of overlap between the "My Tasks" card and the "Current Statuses" card. What's your thoughts about consolidating those views and then having the dropdown which allows the users to filter/sort by? Potential options: "With State" , "With CMS", "Denied", "Approved" (similar to the V2?).

- I like have the actions buttons next to the line/dashboard items. May need to think through if there are other potential items or if there a 1-1 correlation between the status and the potential action items.

- Small thing -- instead of a text link, it would be nice to have "Approval Letter" as a button like the other items (for consistency).

- Did we ever work out how to group APD and Contracts that were submitted at the same time for review (or are we thinking that would be treated independently). I'm reviewing our conversation from #383 and I don't think we ever documented it there (but I remember talking about it)

Priority and Due Date are great ideas:

- With priority, not sure who/where we would set that (so maybe worth figuring out how that would work). For MVP, I would be OK if that wasn't in here.

- Due Date is something we'd have to think about too. Would it be driven by the regulatory mandated due dates or by due dates built into the workflow of the submission. For MVP, The regulations for due dates only exists in E&E and MMIS, but we can use those as a "soft" guide for HITECH work and then allow us to scale if/when we take on those program areas.

- Recent Activity and Upcoming Events are :+1:  Of course, we'll have to figure out how to populate them...

Let me know if you want to talk through anything or if we need to develop any additional documentation to help with this.

**Comment by lauraponce** [comment]
Thanks so much for the feedback, @jeromeleecms this is all super helpful! 👍I'll try to respond to all your notes/questions...

There is going to be some overlap between tasks and statuses, but the main difference is there will be fewer tasks, and the statuses will show the full history. The idea behind the tasks was to pull out the immediate action items so that it’s obvious what they need to do right now; it's essentially their work queue. The idea behind the statuses is more passive: state users would be using that to check what the current state is.

I put the approval letter in as a link instead of a button because all of the buttons have calls to action  (respond, open, start, etc) but this one is actually just a link to the letter as opposed to a call to action. Does it seem to work with having that context?

It came up in an earlier round of research that a state user wanted to see the contracts associated with the APDs, but I think it’s worth showing this version to a few more people to see what the consensus is there because it would change up the design and would likely involve some strategic hiding and seeking of the content.

For the priority, the way I pictured it working is that if it’s ever a “waiting for state response” it would be high since people down the line are potentially held up by that. If it’s something that is in progress or not started and due soon, it would be high. If it’s not started or in progress and not due soon, medium or low. And etc. 

That’s helpful to know about the dates/deadlines! I’ll let you handle what drives those dates 🙂The soft guide sounds like a great start. 

Recent activity will be the running stream of any activity (things like comments, changes to the doc, etc). We’ll have to think through events a little more, but it’s mostly for important deadlines and things like that.

**Comment by jeromeleecms** [comment]
Thanks @lauraponce! Works for me, do we need to spin up any issues to work through any of these items? Also, did you sync up with @quarterback about his work on the landing page? 

For the approval letter link -- that's fine, it just doesn't line up as neatly with the other buttons :)

At the end of day, I agree, let's get more feedback!

**Comment by lauraponce** [comment]
Created #722 for the dev work on this!

**Comment by NAretakis** [comment]
Perfecto!
